src/app/containers/StoryPromo/index.jsx

```diff
@@ -21,7 +21,7 @@ import MediaIndicatorContainer from './MediaIndicator';
 import isTenHoursAgo from '#lib/utilities/isTenHoursAgo';
 import IndexAlsosContainer from './IndexAlsos';
 
-const PROMO_TYPES = ['top', 'regular', 'leading'];
+const PROMO_TYPES = ['top', 'regular', 'leading', 'recommendation'];
```

Comment:
These promo types are used to determine which layout to use. e.g.:
Story Promo of type 'leading' and 'regular':
<img width="800" alt="Screenshot story promo - leading   regular" src="https://user-images.githubusercontent.com/3028997/80574116-24791b00-89f9-11ea-87ac-abc9562b28b9.png">

Story Promo of type 'top':

<img width="800" alt="Screenshot story promo - top" src="https://user-images.githubusercontent.com/3028997/80574111-2347ee00-89f9-11ea-831e-82cc463a44c9.png">

I think it may be better to pass in another prop instead of adding 'recommendation' as a promo type, since it'd mean you cannot follow the 'regular' layout for Story Promos for recommendations.

Reply:
As the logic to decide whether the heading should be overridden with a `div` should be based on the component the story promo is placed within, and the Recommendations container has not been built yet which is when we will need the override, should I wait until that component is built, then pass in a prop from there?